test/cases/testdata/time/test-time-0970.yaml

```diff
@@ -0,0 +1,96 @@
+cases:
+- data:
+    a:
```

Comment:
I think you can remove the boilerplate data row 3-80?

Reply:
Yes, done